Platform/ADLINK/ComHpcAltPkg/ComHpcAlt.dsc

```diff
@@ -0,0 +1,444 @@
+#
+################################################################################
+[PcdsFeatureFlag.common]
+  #
+  # Activate AcpiSdtProtocol
+  #
+  gEfiMdeModulePkgTokenSpaceGuid.PcdInstallAcpiSdtProtocol|TRUE
+
+  #
+  # Flag to indicate option of using default or specific platform Port Map table
+  #
+  gAmpereTokenSpaceGuid.PcdPcieHotPlugPortMapTable.UseDefaultConfig|FALSE
+
+[PcdsFixedAtBuild]
+
+  gAmpereTokenSpaceGuid.PcdPcieHotPlugGpioResetMap|0x3F
+
+  #
+  # Setting Portmap table
+  #
+  #   * Elements of array:
+  #     - 0:  Index of Portmap entry in Portmap table structure (Vport).
+  #     - 1:  Socket number (Socket).
+  #     - 2:  Root complex port for each Portmap entry (RcaPort).
+  #     - 3:  Root complex sub-port for each Portmap entry (RcaSubPort).
+  #     - 4:  Select output port of IO expander (PinPort).
+  #     - 5:  I2C address of IO expander that CPLD backplane simulates (I2cAddress).
+  #     - 6:  Address of I2C switch between CPU and CPLD backplane (MuxAddress).
+  #     - 7:  Channel of I2C switch (MuxChannel).
+  #     - 8:  It is set from PcieHotPlugSetGPIOMapCmd () function to select GPIO[16:21] (PcdPcieHotPlugGpioResetMap) or I2C for PCIe reset purpose.
+  #     - 9:  Segment of root complex (Segment).
+  #     - 10: SSD slot index on the front panel of backplane (DriveIndex).
+  #
+  #   * Caution:
+  #     - The last array ({ 0xFF, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0xFF }) require if no fully structured used.
+  #     - Size of Portmap table: PortMap[MAX_PORTMAP_ENTRY][sizeof(PCIE_HOTPLUG_PORTMAP_ENTRY)] <=> PortMap[96][11].
+  #   * Example: Bellow configuration is the configuration for Portmap table of Mt. Jade 2U platform.
+  #
+  gAmpereTokenSpaceGuid.PcdPcieHotPlugPortMapTable.PortMap[0]|{ 0xFF, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0xFF }       # Require if no fully structure used
+
+  gAmpereTokenSpaceGuid.PcdSmbusI2cBusSpeed|100000
+
+!ifdef $(FIRMWARE_VER)
+  gEfiMdeModulePkgTokenSpaceGuid.PcdFirmwareVersionString|L"$(FIRMWARE_VER)"
+!endif
+
+[PcdsFixedAtBuild.common]
+  #
+  # Platform config UUID
+  #
+  gAmpereTokenSpaceGuid.PcdPlatformConfigUuid|"$(PLATFORM_CONFIG_UUID)"
+
+  gEfiMdeModulePkgTokenSpaceGuid.PcdSmbiosVersion|0x307
+
+  # Clearing BIT0 in this PCD prevents installing a 32-bit SMBIOS entry point,
+  # if the entry point version is >= 3.0. AARCH64 OSes cannot assume the
+  # presence of the 32-bit entry point anyway (because many AARCH64 systems
+  # don't have 32-bit addressable physical RAM), and the additional allocations
+  # below 4 GB needlessly fragment the memory map. So expose the 64-bit entry
+  # point only, for entry point versions >= 3.0.
+  gEfiMdeModulePkgTokenSpaceGuid.PcdSmbiosEntryPointProvideMethod|0x2
+
+  gAmpereTokenSpaceGuid.PcdSmbusI2cBusNumber|0x07
+  gAmpereTokenSpaceGuid.PcdRtcBusNumber|4
+
+  #
+  # Increasing the maximum size of capsule is to cover ARM Trusted Firmware binaries
+  #
+  gEfiMdeModulePkgTokenSpaceGuid.PcdMaxSizeNonPopulateCapsule|0xE00000

```

Comment:
Already exist in AmpereAltraPkg.dsc.inc with the same setting. I think we can drop this